app/views/comments/_answercomments.html.erb

```diff
@@ -12,6 +12,26 @@
     <div id="answer-comment-details" class="right card-action">
       <p>Author: <%= comment.find_user %> Updated: <%= comment.updated_at.strftime("%D at %r") %></p><br>
     </div>
+    <% if current_user && !comment.upvotes.where(creator: current_user.id).exists? %>
```

Comment:
I actually added this current_user piece when I was fixing merge conflicts earlier because when a guest visited the page, current_user was nil and everything exploded.